Core/src/org/sleuthkit/autopsy/geolocation/MapPanel.form

```diff
@@ -56,10 +56,10 @@
                 <Property name="orientation" type="int" value="1"/>
                 <Property name="paintTicks" type="boolean" value="true"/>
                 <Property name="snapToTicks" type="boolean" value="true"/>
+                <Property name="inverted" type="boolean" value="true"/>
                 <Property name="minimumSize" type="java.awt.Dimension" editor="org.netbeans.beaninfo.editors.DimensionEditor">
                   <Dimension value="[35, 100]"/>
                 </Property>
-                <Property name="opaque" type="boolean" value="false"/>
```

Comment:
Why was this changed? It looks much better as opaque.  That was an intentional design decision.  What was the thought behind changing it?

Reply:
It wasn't consciously changed. This happened automatically by the editor. I'm looking into changing it currently.